analyses/molecular-subtyping-HGG/07-HGG-molecular-subtyping-combine-table.Rmd

```diff
@@ -0,0 +1,529 @@
+         Kids_First_Biospecimen_ID_RNA) %>%
+  mutate(molecular_subtype = "HGG_H3_G35")
+```
+
+### IDH mutant
+
+> * These tumors contain IDH1 R132H mutations
+> * Co-occurring lesions include: TP53 mutations; P73-AS1 promoter methylation and downregulation
+> * High expression of FOXG1 and OLIG2
+> * Mutually exclusive lesions: chr7 gain and chr10 loss
+
+```{r}
+idh_df <- all_data_df %>%
+  filter(IDH1_mutation == "p.R132H")
+idh_df
+```
+
+In these instances, the broad copy data look consistent with the description, all tumors have _TP53_ mutations, and the _TP73-AS1_ expression is consistent with downregulation.
+
+Make a biospecimen-centric table.
+
+```{r}
+idh_df_biospecimen <- all_data_df %>%
+  filter(IDH1_mutation == "p.R132H") %>%
+  select(Kids_First_Participant_ID,
+         sample_id,
+         Kids_First_Biospecimen_ID_DNA,
+         Kids_First_Biospecimen_ID_RNA) %>%
+  mutate(molecular_subtype = "HGG_IDH")
+```
+
+
+### H3.3 and IDH wildtype
+
+> * High-grade gliomas absent of H3F3A and IDH mutations
+> * Defining lesions: MYCN, PDGFRA amplification, TP53 and TERT mutations
+
+```{r}
+wildtype_df <- all_data_df %>%
+  filter(H3F3A.K28M == "No",
+         HIST1H3B.K28M == "No",
+         H3F3A.G35R == "No",
+         H3F3A.G35V == "No",
+         !grepl("IDH1", relevant_coding_mutations)) %>%
+  select(Kids_First_Participant_ID,
+         sample_id,
+         relevant_coding_mutations,
+         TERT_variant_classification,
+         MYCN_focal_status,
+         PDGFRA_focal_status) %>%
+  distinct()
+
+wildtype_df %>% arrange(MYCN_focal_status, PDGFRA_focal_status)
+```
+
+This is not particularly straightforward, so I will leave that out for now.
+Possible next steps: only those with amplifications or gains in the two mentioned genes?
+
+### 1p/19q co-deleted oligodendrogliomas
+
+> * Co-deletion of chr 1p and 19q (LOH, loss of heterozygosity of both) results in translocation t(1p;19q)
+> * Nearly all have a co-occurring IDH1 mutation (R132 or R172)
+> * Other co-occurring lesions include: TERT promoter, CIC, and FUBP1 mutations
+> * Mutually exclusive of TP53 and ATRX mutation
+> * Typically occurs in adult tumors
+
+Let's first filter by the broad arm data and see what we get.
+
+```{r}
+all_data_df %>%
+  filter(`1p` == "loss" | `19q` == "loss")
+```
+
+We don't see the _IDH1_ mutations we would expect and we do see an H3 K28M mutation, so we will leave this off as well for the moment and in a subsequent notebook we'll address [`AlexsLemonade/OpenPBTA-analysis#435 (comment)`](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/435#issuecomment-576898275):
+
+> We expect very few, if any of these (maybe 1 when we looked by CNV), so I think maybe we can approach this cohort by starting to look for 1p/19q codeletions in all samples + IDH mutations, and if we see any, we may manually want to check the CNV plots just to confirm.
+
+### Samples without histone mutations that have _BRAF_ V600 mutations
```

Comment:
Can we add a visualization to highlight these samples in a tsne or umap clustering of all of the data by broad histology? If they cluster with the LGAT, then I think we can re-categorize them. Maybe this is a separate notebook.

Reply:
I think this is a separate notebook and I'm most likely going to file a separate ticket that's focused just on this visualization.